js/event.ts

```diff
@@ -0,0 +1,282 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import * as domTypes from "./dom_types";
+
+export const eventAttributes = new WeakMap();
```

Comment:
Just a nit, but some comments would help the clarity here.

Reply:
all set